src/pipeline.py

```diff
@@ -1,10 +1,17 @@
-from dplypy import DplyFrame
+from src.dplypy import DplyFrame
 
-def query(query_str: str): # summary_type: e.g. mean, variance, other descriptive stats, num rows, 
+
+def query(query_str: str):  # summary_type: e.g. mean, variance, other descriptive stats, num rows,
     return lambda d1: DplyFrame(d1.pandas_df.query(query_str))
 
-def apply(func, axis=0, **kwargs): # TODO: CSC: use kwargs... couldn't quickly figure out how to do it
-    return lambda d1: DplyFrame(d1.pandas_df.apply(func=func, axis=axis)) 
+
+def apply(func, axis=0, **kwargs):  # TODO: CSC: use kwargs... couldn't quickly figure out how to do it
+    return lambda d1: DplyFrame(d1.pandas_df.apply(func=func, axis=axis))
+
+
```

Comment:
Remove extra new line

Reply:
I looked up PEP8 and found that top-level functions are separated by two blank lines, and class methods are separated by one blank line. Perhaps it's better to leave it as two?

Reply:
I see, I see. Yeah, let's follow the PEP8 guidelines. Maybe we run linter each as a pre-commit hook?